components/tidb_query_vec_executors/src/index_scan_executor.rs

```diff
@@ -344,148 +342,163 @@ impl IndexScanExecutorImpl {
         Ok(())
     }
 
-    // Process index values that contain common handle flags.
-    // NOTE: If there is no restore data in index value, we need to extract the index column from the key.
-    // 1. Unique common handle in new collation
-    // |  Layout: 0x00 | CHandle Flag | CHandle Len | CHandle       | RestoreData
-    // |  Length: 1    | 1            | 2           | size(CHandle) | size(RestoreData)
-    //
-    // 2. Unique common handle in old collation
-    // |  Layout: 0x00 | CHandle Flag | CHandle Len | CHandle       |
-    // |  Length: 1    | 1            | 2           | size(CHandle) |
-    fn process_unique_common_handle_kv(
+    // Process index values that are in old collation but don't contain common handles.
+    // NOTE: We should extract the index columns from the key first, and extract the handles from value if there is no handle in the key.
+    // Otherwise, extract the handles from the key.
+    fn process_old_collation_kv(
```

Comment:
I prefer to name this function as `process_unconfigured_kv` and named the `process_configured_kv` which fit the means of the option part of the value.

Reply:
`process_old_collation_kv` is an old name, I did not change it. So I leave it here. `old collation` is also used in [docs](https://docs.google.com/document/d/1Co5iMiaxitv3okJmLYLJxZYCNChcjzswJMRr-_45Eqg/edit?usp=sharing)  for value collation.